docs/operations-manual.md

```diff
@@ -279,9 +267,7 @@ cd ./helm
 helm install metadig-controller ./metadig-controller --namespace metadig --version=1.0.0
 ```
 
-The metadig-controller helm chart contains the Java property file ("metadig.properties") that is used by all metadig k8s applications. If you
-wish to change any of the configuration values, this property file can be edited locally and then restarted with helm. Since all of the metadig 
-applications mount this configuration file as a k8s volume, they will need to be restarted after any configuration change.
+The metadig-controller helm chart contains the Java property file ("metadig.properties") that is used by all metadig k8s applications. If you wish to change any of the configuration values, this property file can be edited locally and then restarted with helm. Since all of the metadig  applications mount this configuration file as a k8s volume, they will need to be restarted after any configuration change.
```

Comment:
One way to handle this is to use a hash of the property file as a metadata field in the helm deployment descriptor to automatically roll pods when the file changes. This is particularly useful when the property file is deployed using a ConfigMap. See https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments